### PR TITLE
now generating deep copies of the collection with the objects triggering the event

### PR DESCRIPTION
**Pull request by gianipez**
tagging: @brownd1978 @rlcee

### CODE REVIEW

**Comment by gianipez** [comment]
Once this PR will be in, we will have to re-generate a reference file as there will be additional collections producing histograms

File: Validation/trigger.fcl [hunk 1]
```diff
@@ -42,3 +42,33 @@ physics.trigger_paths : [@sequence::Trig_valMenu.trigger_paths]
 physics.producers.EWMProducer.SpillType : 1
 #include "Production/JobConfig/common/epilog.fcl"
 #include "Production/JobConfig/reco/drop_trigger.fcl"
+
+
+physics.producers.aprHelixTriggerInfoMerger.ntupMode                : 1
```
Comment (brownd1978) [inline Validation/trigger.fcl:47]: Didnt the name of that parameter change? Is a change in this PR to follow that needed?

**Review by brownd1978 — COMMENTED** [review]
Are these deep copies related to the increased resource usage seen in the nightly trigger validation?

File: Validation/trigger.fcl (hunk 1 above)
Comment (gianipez) [reply Validation/trigger.fcl]: thanks for spotting this. I simply forgot the last `git push`. Now everything should be fine. I tested it locally successfully.

**Review by brownd1978 — APPROVED** [review]

**Pull request merged**